src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +87,24 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * void fun1() {}    // violation, 4 whitespaces before the comment starts
+ * void fun2() { return; }  /* violation here, 2 whitespaces before the comment starts *&#47;
+ *
+ * /* violation, 2 whitespaces after the comment ends *&#47;  int a;
+ *
+ * String s; /* OK *&#47;
+ *
+ * /**
+ * *  This is a Javadoc comment
+ * *&#47;    int b; // violation, 4 whitespaces after the javadoc comment ends
+ *
+ * float f1; // OK
+ * float f2; /**
+ *            * OK
+ *            *&#47;
```

Comment:
please move javadoc before field

Reply:
Then should i show the line
"public class test {"
in the example in the website as well?

Reply:
I see no javadoc at all now

Reply:
It's above all the code now. Please check

Reply:
It's before the field as you suggested @strkkk